android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,4 @@
+<resources>
+    <string name="app_name">OpenBlack</string>
+</resources>
```

Comment:
@handsomematt how would you like the app name to appear in an android app? In regards to capitalization